<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieCaptureDialog</class>
 <widget class="QDialog" name="MovieCaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie Recording</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Destination video:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horisontalLayout">
     <item>
      <widget class="QLineEdit" name="destinationEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>FPS (frames per second)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSpinBox" name="fpsSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fpsSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>destinationEdit</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>fpsSpin</tabstop>
  <tabstop>fpsSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
